documentation/src/main/antora/modules/ROOT/pages/tutorial-create-my-first-component.adoc

```diff
@@ -0,0 +1,165 @@
+= Creating your first component
+:page-partial:
+
+This tutorial walks you through all the required steps to get started with *Talend Component Kit*, from the creation of a simple component to its integration into *Talend Studio*.
+
+The component created in this tutorial is a simple output component that only reads the data it receives and displays it in the logs, along with an extra information entered by a user.
+
+Once the prerequisites completed, this tutorial should take you about 20 minutes.
+
+image::tutorial_build_job_run.png[Job run]
+
+== Prerequisites
+As this tutorial aims at helping you to create your very first component, it also covers some of the installation steps that are specific to Talend Component Kit to get your development environment ready. But first, make sure to:
+
+- Download and install a Java JDK 1.8 or greater.
+// Download and install Maven 3.5.x.
+- Download and install Talend Studio. For example, from link:https://sourceforge.net/projects/talend-studio[Sourceforge].
+- Download and install https://www.jetbrains.com/idea/download[IntelliJ].
+- Download the Talend Component Kit plugin for IntelliJ. The detailed installation steps for the plugin are available in xref:installing-talend-intellij-plugin.adoc[this document].
+
+== Generating a simple component project
+The first step in this tutorial is to generate a project containing a simple output component using the starter.
+
+. Start IntelliJ and create a new project. In the available options, you should see *Talend Component*. +
++
+image::intellij_new_component_project.png[New project]
+. Make sure that a *Project SDK* is selected. Then, select *Talend Component* and click *Next*. +
+The Talend Component Kit Starter opens.
+. Enter the project details. The goal here is to define the component and project metadata. Change the default values as follows: +
++
+image::intellij_tutorial_project_metadata.png[Project metadata]
+- The *Component Family* and the *Category* will be used later in Talend Studio to find the new component.
+- The project metadata are mostly used to identify the project structure. A common practice is to replace 'company' in the default value by a value of your own, like your domain name. +
+. Once the metadata is filled, select *ADD A COMPONENT*. A new screen is displayed in the starter that lets you define the generic configuration of the component.
```

Comment:
can you please replace the word *starter* with the full application name *Talend Component Kit Starter *. 
it's useful for indexing the tutorial in search engins.